angular-date-picker.js

```diff
@@ -133,7 +135,14 @@
 
                 $scope.pickDay = function (evt) {
                     var target = angular.element(evt.target);
-
+                    
+                    if (target.hasClass('-leadingDay')) {
+                        $scope.month--;
+                    }
+                    if (target.hasClass('-trailingDay')) {
+                        $scope.month++;
+                    }
+                    
                     if (target.hasClass('-selectable')) {
                         var day = parseInt(target.text(), 10);
```

Comment:
But maybe the month offseting can be moved here and simply based on a data-month-offset attribute that either non-existent, 1 or -1:

```
var monthOffset = target.data('month-offset');

if (monthOffset) {
    $scope.month += parseInt(monthOffset, 10);
}
```